README.md

```diff
@@ -6,6 +6,11 @@ List of GitHub profiles that have awesome customisation, that you can use for in
 
 Feel free to submit a pull request with your suggestions of awesome github profiles.
 
+[![made with love](https://forthebadge.com/images/badges/built-with-love.svg)](https:// github.com/EddieHubCommunity)
+[![made with markdown](https://forthebadge.com/images/badges/made-with-markdown.svg)](https://forthebadge.com)
+[![open source](https://forthebadge.com/images/badges/open-source.svg)](https://forthebadge.com)
+[![check it out](https://forthebadge.com/images/badges/check-it-out.svg)](https://forthebadge.com)
+
```

Comment:
I would also adjust the links here.
1. no link at all
2. other link than forthebadge